command/common/command_list.go

```diff
@@ -36,6 +36,8 @@ type commandList struct {
 	AllowSpaceSSH                      v2.AllowSpaceSSHCommand                      `command:"allow-space-ssh" description:"Allow SSH access for the space"`
 	Api                                v2.ApiCommand                                `command:"api" description:"Set or view target api url"`
 	Apps                               v2.AppsCommand                               `command:"apps" alias:"a" description:"List all apps in the target space"`
+	AllowNetworkAccess                 v3.AllowNetworkAccessCommand                 `command:"allow-network-access" description:"Allow direct network traffic from one app to another"`
+	RemoveNetworkAccess                v3.RemoveNetworkAccessCommand                `command:"remove-network-access" description:"Remove direct network traffic from one app to another"`
```

Comment:
Outside of the top V3/experimental section. We try and keep the command list in alphabetical order.